metrics.go

```diff
@@ -135,6 +138,7 @@ type RealtimeMetrics struct {
 	Aggregated Metrics               `json:"aggregated"`
 	ByHost     map[string]Metrics    `json:"by_host,omitempty"`
 	ByDisk     map[string]DiskMetric `json:"by_disk,omitempty"`
+	ByNet      map[string]NetMetrics `json:"by_net,omitempty"`
```

Comment:
Do you mean "By NIC"? Not really sure what "net" refers to here.

What are they added by?

Reply:
OK, so you are adding these by Host.

You should add data to `Metrics` struct and not here.

This will automatically populate `ByHost` since you are setting `ByHost: true`